### PR TITLE
Mpl toolkit fix for zoomed_inset_axes

### PR DESCRIPTION
**Pull request by Tuan333**
## PR Summary
A small fix for #8589. Test cases provided.

## PR Checklist

- [x] Has Pytest style unit tests
- [x] Code is PEP 8 compliant 
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/whats_new.rst if major new feature
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

**Comment by Tuan333** [comment]
Note: The errors on Appveyor don't seem to reappear in my local machine or an Appveyor built from my forked repo.

File: lib/matplotlib/tests/test_axes.py [hunk 1]
```diff
@@ -5149,3 +5149,21 @@ def test_twinx_knows_limits():
...
+
+
+@image_comparison(baseline_images=['zoomed_axes',
+                                   'inverted_zoomed_axes'],
+                  extensions=['png'])
+def test_zooming_with_inverted_axes():
+    fig, ax = plt.subplots()
+    ax.plot([1, 2, 3], [1, 2, 3])
+    ax.axis([1, 3, 1, 3])
+    from mpl_toolkits.axes_grid1.inset_locator import zoomed_inset_axes
```
Comment (QuLogic) [inline lib/matplotlib/tests/test_axes.py]: If it uses `mpl_toolkits`, it doesn't belong in this file. It should go somewhere in `lib/mpl_toolkits/tests`.

Comment (Tuan333) [inline lib/matplotlib/tests/test_axes.py]: sure. thanks for the pointer

**Comment by Tuan333** [comment]
As with last time, the appveyor build fails, but i can't reproduce. If someone could rebuild to see if it still fails, i'll be thankful

File: lib/mpl_toolkits/tests/test_axes_grid1.py [hunk 2]
```diff
@@ -154,3 +154,21 @@ def get_demo_image():
...
+
+
+@image_comparison(baseline_images=['zoomed_axes',
+                                   'inverted_zoomed_axes'],
+                  extensions=['png'])
+def test_zooming_with_inverted_axes():
+    fig, ax = plt.subplots()
+    ax.plot([1, 2, 3], [1, 2, 3])
+    ax.axis([1, 3, 1, 3])
+    from mpl_toolkits.axes_grid1.inset_locator import zoomed_inset_axes
```
Comment (QuLogic) [inline lib/mpl_toolkits/tests/test_axes_grid1.py]: This is already imported at the top of the file.

**Comment by Tuan333** [comment]
fixed test case by @QuLogic  comment and squashed commit. If appveyor build still fails (hopefully it won't), someone please rebuild it in my stead, something is quite wrong with my GitHub settings. Thank you.

**Comment by QuLogic** [comment]
Probably should have corrected the commit message a bit, though...

**Pull request merged**